01-main/packages/firefox

```diff
@@ -1,6 +1,11 @@
 DEFVER=1
-CODENAMES_SUPPORTED="mantic lunar jammy focal bionic xenial"
+# Per https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu/dists/noble/main/ :
+ARCHS_SUPPORTED="amd64 arm64 armhf i386 ppc64el riscv64 s390x"
+# Per https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu/dists/ :
+# focal groovy gutsy hardy hirsute impish intrepid jammy jaunty kinetic lunar mantic noble trusty xenial zesty
+CODENAMES_SUPPORTED="focal jammy kinetic lunar mantic"
+APT_LIST_NAME="mozilla.org"
```

Comment:
This is not appropriate for the ppa, which is a separate repository / key from the apt repository so we need to have a list for ppa and a list for the main apt repo
```suggestion
APT_LIST_NAME="ppa.mozilla.org"
```